### DISCUSSION TITLE
Cloud Reliability

### DISCUSSION THREAD

**Discussion by jonnymccullagh** [openobserve/openobserve#3541] [General]
Hi, I have having some issues with the reliability of the SaaS offering. Dashboards aren't loading - I have attached screenshot of the console - there is an Axios error you may be able to use.
Also I am getting CORS errors for: https://monitor.dev.zinclabs.dev/rum

My main question is would I be better self hosting it ? 

<img width="365" alt="image" src="https://github.com/openobserve/openobserve/assets/5209315/1ed11260-30b3-4b08-a838-24d1a860d4b4">

**Comment by prabhatsharma** [comment]
You can safely ignore these console errors. It's part of our monitoring stack for the cloud service itself.

what are the errors around dashboards? Do you see any errors there?
